packages/connectors/connector-helper/src/ethereum.ts

```diff
@@ -4,19 +4,22 @@ import type {
 	EthereumProviderConnectionResult,
 } from "@rarible/connector"
 import { EthereumWallet } from "@rarible/sdk-wallet"
-import { Web3Ethereum } from "@rarible/web3-ethereum"
-import Web3 from "web3"
+import { Web3Ethereum, Web3 } from "@rarible/web3-ethereum"
 import type { EVMBlockchain } from "@rarible/sdk-common"
 import { getBlockchainFromChainId } from "@rarible/protocol-ethereum-sdk/build/common"
+import { Web3v4Ethereum, Web3 as Web3v4 } from "@rarible/web3-v4-ethereum"
 import type { IWalletAndAddress } from "./wallet-connection"
 
+/**
+ * Use this function for wrapping web3 v1 instance
+ * If you need to use web3 v4 use "mapEthereumWeb3v4Wallet" function
+ */
 export function mapEthereumWallet<O>(
```

Comment:
it must be separated on different files since user will have just 1 version web3-ethereum OR web3-v4-ethereum and on the end it will lead to throwing exception on build time